app/sequencescape/sequencescape/api/v2/tube.rb

```diff
@@ -51,6 +51,11 @@ def self.find_all(params)
 
   delegate :requests_as_source, to: :receptacle
 
+  # Mocked out for the time being
```

Comment:
This comment was carried here from the V1 Tube model.  I don't know how long "the time being" is, but I was trying to avoid making changes to how this works compared to V1.